### PR TITLE
FAQ (PDFNET-57969)

### PR DESCRIPTION
**Pull request by alevsh**

### CODE REVIEW

**Comment by lightway82** [comment]
```
using (Document mergedDocuments = Document.MergeDocuments("input1.pdf", "input2.pdf", `"input3.pdf"))`
{
     mergedDocuments.Save(outputPdfFile);
 }
```

File: net/faq/_index.md [hunk 1]
```diff
@@ -5,10 +5,302 @@ type: docs
...
+```csharp
+// Load the tagged PDF document
+Document pdfDoc = new Document("tagged.pdf");
+
+// Validate for PDF/UA compliance
+pdfDoc.Validate("validation-log.xml", PdfFormat.PDF_UA_1);
+```
+
+## How to implement regex for TextSegmentAbsorber?
+To use regex with the `TextSegmentAbsorber` class in Aspose.PDF for .NET, you can follow this example:
```
Comment (AlexBiriukov) [inline net/faq/_index.md:242]: Here and below: TextFragmentAbsorber, not TextSegmentAbsorber

**Review by AlexBiriukov — COMMENTED** [review]
Some things need to be corrected.

File: net/faq/_index.md [hunk 2]
```diff
@@ -5,10 +5,296 @@ type: docs
...
...d image to a page object. Follow the <a href="/pdf/net/add-backgrounds/">link</a> to show an example of code for adding a background.</p></div></div><div itemscope="" itemprop="mainEntity" itemtype="https://schema.org/Question" class="faq-question"><h3 itemprop="name" class="faq-q">PDF File Security: How to Secure PDF File or a Document?</h3><div itemscope="" itemprop="acceptedAnswer" itemtype="https://schema.org/Answer"><p itemprop="text" class="faq-a">Aspose.PDF for .NET allows you to set privileges on an existing PDF file. You can use the Encrypt method of the Document object to encrypt a PDF file. Also, you can decrypt PDF files using the owner's password or change the password of a PDF file. For more details visit the documentation <a href="/pdf/net/digitally-sign-pdf-file/">page</a>.</p></div></div><div itemscope="" itemprop="mainEntity" itemtype="https://schema.org/Question" class="faq-question"><h3 itemprop="name" class="faq-q">Where are your .NET Examples?</h3><div itemscope="" itemprop="acceptedAnswer" itemtype="https://schema.org/Answer"><p itemprop="text" class="faq-a">You can check all of them on <a href="https://github.com/aspose-pdf">GitHub</a>.</p></div></div></div>
```
Comment (andruhovski) [inline net/faq/_index.md:14]: It's not an HTML code. It's a structured data entity FAQPage. Please restore!

**Review by andruhovski — CHANGES_REQUESTED** [review]

File: net/faq/_index.md [hunk 3]
```diff
@@ -5,10 +5,299 @@ type: docs
...
+
+```csharp
+// Load the tagged PDF document
+Document pdfDoc = new Document("tagged.pdf");
+
+// Validate for PDF/UA compliance
+pdfDoc.Validate("validation-log.xml", PdfFormat.PDF_UA_1);
+```
+
+## How to implement regex search for TextFragemntAbsorber?
```
Comment (BaronRastignak) [inline net/faq/_index.md:236]: Typo: TextFrag**me**ntAbsorber

File: net/faq/_index.md [hunk 4]
```diff
@@ -5,10 +5,299 @@ type: docs
...
+
+- Set the `DefaultFontName` property to define a substitute font.
+- Use `Convert` to ensure the document meets PDF/A standards.
+
+```csharp
+// Load the document
+Document pdfDoc = new Document("input.pdf");
+
+// Set default font for substitution
+pdfDoc.FontUtilities.DefaultFontName = "Arial";
```
Comment (BaronRastignak) [inline net/faq/_index.md:280]: There is no such property here. The correct snippet may look like:
```csharp
using (Document pdfDoc = new Aspose.Pdf.Document("input.pdf"))
{
    PdfFormatConversionOptions options = new PdfFormatConversionOptions("conversion-log.xml", PdfFormat.PDF_A_1B, ConvertErrorAction.Delete);
    // Substitute all missing fonts with the default one
    options.FontEmbeddingOptions.UseDefaultSubstitution = true;

    // Convert to PDF/A
    pdfDoc.Convert(options);

    // Save as PDF/A
    pdfDoc.Save("output-pdfa.pdf");
}
```

File: net/faq/_index.md [hunk 5]
```diff
@@ -5,10 +5,299 @@ type: docs
...
+pdfDoc.Convert("conversion-log.xml", PdfFormat.PDF_A_1B, ConvertErrorAction.Delete);
+
+// Save as PDF/A
+pdfDoc.Save("output-pdfa.pdf");
+```
+
+## Does Aspose.PDF for .NET support Linux?
+Yes, Aspose.PDF for .NET supports running on Linux environments. You can use the .NET Core version or later, which is cross-platform and can be used on Windows, macOS, and Linux.
+
+## Does Aspose.PDF for .NET support .NET 2.0, 3.5, and 4.0 frameworks?
```
Comment (BaronRastignak) [inline net/faq/_index.md:292]: We've dropped support for at least .NET Framework 2.0 and 3.5, haven't we?

**Review by BaronRastignak — COMMENTED** [review]
One more quite frequent PDF/A related question:

I see errors in a PDF/A conversion log. Does it mean that the document wasn't converted?

Answer: No, Aspose.PDF logs all problems it encountered, including the ones that were automatically fixed. If all entries in the log are marked as Convertable="True", it means that all problems were fixed, and the document was successfully converted. Only the entries with Convertable="False" indicate the conversion failure.

File: net/faq/_index.md [hunk 6]
```diff
@@ -5,10 +5,518 @@ type: docs
...
+        </h3>
+    </div>
+    <div itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
+        <p itemprop="text" class="faq-a">
+Aspose.PDF for .NET does not support the following .NET frameworks:
+
+- .NET 2.0
+- .NET 3.5
+
+But you can use Aspose.PDF for .NET with any of these frameworks in your project. The library provides a consistent API across the different .NET versions, making it easy to migrate your code between frameworks as needed.
```
Comment (AnHolub) [inline net/faq/_index.md:442]: However, you can use Aspose.PDF for .NET with any of these frameworks for your project.

File: net/faq/_index.md [hunk 7]
```diff
@@ -5,10 +5,518 @@ type: docs
...
+    </div>
+
+    <div itemscope itemprop="mainEntity" itemtype="https://schema.org/Question" class="faq-question">
+        <h3 itemprop="name" class="faq-q">
+I see errors in PDF/A conversion log. Does it mean that the document wasn't converted successfully?
+        </h3>
+    </div>
+    <div itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
+        <p itemprop="text" class="faq-a">
+No, Aspose.PDF logs all problems it encountered, including the ones that were automatically fixed. If all entries in the log are marked as Convertable="True", it means that all problems were fixed, and the document was successfully converted. Only the entries with Convertable=False indicate the conversion failure.
```
Comment (AnHolub) [inline net/faq/_index.md:415]: If all entries in the log are marked as Convertable="True, " all problems were fixed, and the document was successfully converted. Only the entries with Convertable=False indicate the conversion failure.

**Review by AnHolub — APPROVED** [review]

**Review by AlexBiriukov — APPROVED** [review]
No new comments.

**Pull request merged**